src/audio/module_adapter/module_adapter_ipc4.c

```diff
@@ -67,15 +67,20 @@ module_ext_init_decode(struct comp_dev *dev, struct module_config *dst,
 				(const struct ipc4_module_init_ext_obj_dp_data *)(obj + 1);
 
 			if (obj->object_words * sizeof(uint32_t) < sizeof(*dp_data)) {
```

Comment:
to the commit message, I understand by "released" you mean it's already a part of the ABI? Maybe better use that wording, since I first thought "released" meant "freed" which was confusing